src/Resources/config/services.yml

```diff
@@ -157,14 +151,14 @@ services:
         class: FriendsOfSylius\SyliusImportExportPlugin\Service\DefaultAddressConcatenation
 
     # generic ResourceProcessors, which can work with just Property-Accessors
-    sylius.processor.tax_categories:
+    sylius.processor.countries:
```

Comment:
What has happened here? 🙀

Reply:
The tax categories processor was moved to the tax categories related file and since its definition is similar to the countries processor, git treats this lines like they were changed, I guess :P